core/src/HIP/Kokkos_HIP_KernelLaunch.hpp

```diff
@@ -147,13 +152,23 @@ struct HIPParallelLaunch<
              block.y, block.z, shmem);
       printf("Pre Launch Error: %s\n", hipGetErrorName(hipGetLastError()));
 
+      // FIXME_HIP -- there is currently an error copying (some) structs
+      // by value to the device in HIP-Clang / VDI
+      // As a workaround, we can malloc the DriverType and explictly copy over.
+      // To remove once solved in HIP
+      DriverType *device;
```

Comment:
oh-my, Still device is an unfortunate name, since we use device as a name for a GPU instance and/or a combination of an execution and a memory space.

Reply:
Does d_driver work?

Reply:
0332a0b